README.md

```diff
@@ -80,7 +99,7 @@ $Env:GYP_DEFINES = 'GTK_Root="C:/Path/To/GTK" jpeg_root="C:/Path/To/libjpeg"'
 [libjpeg-turbo development files]: https://sourceforge.net/projects/libjpeg-turbo/files/2.0.6/libjpeg-turbo-2.0.6-vc64.exe/download
 [Scoop]: https://scoop.sh/
 
-## How To Run
+### How To Run
```

Comment:
Before this, there should be a `Step 6.` to ensure `wsl` is installed. The steps to do this manually are in https://github.com/rancher-sandbox/rd/issues/185#issuecomment-825894149

Reply:
Added link to WSL install directions.